css/style.css

```diff
@@ -61,23 +61,217 @@ body {
 
 /* Center the container */
 .container {
-  width: 960px;
-  margin: auto;
+  width: 100%;
   height: 500px;
+  margin: 0 auto;
+  background: url(../images/header_bg.jpg) no-repeat center top rgb(134,141,157);
+  background-size: cover;
+  float: left;
```

Comment:
:hammer: This isn't necessary for a few reasons. First, a div will automatically be aligned to the left (generally speaking). Second, you have the width set to 100%, so floating either way won't make a difference.

But there _is_ something you can do to the container to finish of the page. You're so close! I see you removed the 960px width. Put that back and see what happens. Everything shrunk and is the the left, correct? There's a nice single-line simple margin trick to get it to be centered on the page. Do you remember? It has to do with **auto**matically setting the left and right margins equally. :)